File: Python Program/LCM.py
#LCM of two numbers    LCM= Least common multiple

print("Enter two numbers")       
a=int(input())          
b=int(input())     
L= a if a>b else b      
while L<=a*b:                       
    if L%a==0 and L%b==0:    
        print("LCM is ",L)    
        break      
    L+=1          
print()   

File: C Program/LCM.c
// To find LCM of number   LCM=Least Common Multiple

#include<stdio.h>
int main()
{
    int a,b,L;
    printf("Enter two number ");
    scanf("%d%d",&a,&b);
    for(L=a>b?a:b;L<=a*b;L++)
       if(L%a==0 && L%b==0)
       break;
    printf("LCM is %d",L);
    printf("\n");
    return 0;
}